### PR TITLE
Allow an existing secret to be used

### PR DESCRIPTION
**Pull request by adinhodovic**
# Pull Request

## What does this PR do?
Adds the functionality to specify an existing secret that has the MEILI_MASTER_KEY defined.

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to MeiliSearch!

### CODE REVIEW

File: charts/meilisearch/README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ This command deploys MeiliSearch on your Kubernetes cluster using the default co
 
 ## Uninstalling the Chart
 
-To uninstall/delete the MeiliSearch` deployment:
+To uninstall/delete the MeiliSearch deployment:
```
Comment (alallema) [inline charts/meilisearch/README.md:34]:
```suggestion
To uninstall/delete the `MeiliSearch` deployment:
```
Good catch but I think one was missing

**Review by alallema — COMMENTED** [review]
Hi @adinhodovic,
Nice addition! I just asked for a little change for the backquote correction.
Thanks for contributing to Meilisearch ❤️

**Review by alallema — APPROVED** [review]
LGTM! 🔥 
Thanks, @adinhodovic for your contribution.

**Comment by adinhodovic** [comment]
> LGTM! fire Thanks, @adinhodovic for your contribution.

Thanks for the quick review!

**Comment by alallema** [comment]
bors merge

**Pull request merged**